test/vinmonopolet.test.js

```diff
@@ -337,6 +379,22 @@ describe('vinmonopolet', function () {
     )
   })
 
+  describe.only('getProductsByStore', () => {
```

Comment:
I think the `only` here is an oversight?

Reply:
Dang it. Thanks